thor/main.py

```diff
@@ -49,12 +51,12 @@
 ]
 
 def rangeAndShift_worker(observations, ephemeris, cell_area=10):
-    
```

Comment:
Sorry about all the whitespace changes! My emacs setup cleans up whitespace on save. Maybe I should write a script to go through the whole repo so we don't constantly look at this crap.

Reply:
No worries, I updated my vscode settings to do the same. I'll take care of the trailing whitespaces in a single commit after I merge this.